requirements.txt

```diff
@@ -24,3 +24,6 @@ launchdarkly-api==2.0.15
 
 # Runtime
 gunicorn==19.9.0
+
+# Datadog
+datadog
```

Comment:
@rmanalan can you please pin this to a specific version?

Reply:
Done.